2022-01-20 16:14:40,185 INFO    MainThread:25636 [wandb_setup.py:_flush():71] setting env: {}
2022-01-20 16:14:40,185 INFO    MainThread:25636 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2022-01-20 16:14:40,186 INFO    MainThread:25636 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\yunjc\_python_jupyter\bupyeonggu\bp_road_crack_detection\_model_test\Pytorch-UNet-master\wandb\run-20220120_161440-1eaxqdo0\logs\debug.log
2022-01-20 16:14:40,186 INFO    MainThread:25636 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\yunjc\_python_jupyter\bupyeonggu\bp_road_crack_detection\_model_test\Pytorch-UNet-master\wandb\run-20220120_161440-1eaxqdo0\logs\debug-internal.log
2022-01-20 16:14:40,186 INFO    MainThread:25636 [wandb_init.py:init():404] calling init triggers
2022-01-20 16:14:40,186 INFO    MainThread:25636 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-20 16:14:40,186 INFO    MainThread:25636 [wandb_init.py:init():460] starting backend
2022-01-20 16:14:40,186 INFO    MainThread:25636 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-01-20 16:14:40,189 INFO    MainThread:25636 [backend.py:ensure_launched():216] starting backend process...
2022-01-20 16:14:40,238 INFO    MainThread:25636 [backend.py:ensure_launched():221] started backend process with pid: 17084
2022-01-20 16:14:40,239 INFO    MainThread:25636 [wandb_init.py:init():469] backend started and connected
2022-01-20 16:14:40,239 INFO    MainThread:25636 [wandb_init.py:init():533] updated telemetry
2022-01-20 16:14:40,263 INFO    MainThread:25636 [wandb_init.py:init():563] communicating current version
2022-01-20 16:14:41,239 INFO    MainThread:25636 [wandb_init.py:init():568] got version response 
2022-01-20 16:14:41,240 INFO    MainThread:25636 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-20 16:14:41,687 INFO    MainThread:25636 [wandb_init.py:init():606] starting run threads in backend
2022-01-20 16:14:43,819 INFO    MainThread:25636 [wandb_run.py:_console_start():1810] atexit reg
2022-01-20 16:14:43,820 INFO    MainThread:25636 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-01-20 16:14:43,820 INFO    MainThread:25636 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-01-20 16:14:43,821 INFO    MainThread:25636 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-20 16:14:43,821 INFO    MainThread:25636 [wandb_init.py:init():633] run started, returning control to user process
2022-01-20 16:14:43,821 INFO    MainThread:25636 [wandb_run.py:_config_callback():956] config_cb None None {'epochs': 5, 'batch_size': 1, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.4, 'amp': True}
2022-01-20 16:14:52,835 INFO    MainThread:25636 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-20 16:14:52,836 INFO    MainThread:25636 [wandb_run.py:_restore():1752] restore
2022-01-20 16:14:53,469 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 873
  total_bytes: 873
}

2022-01-20 16:14:53,782 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 873
  total_bytes: 873
}

2022-01-20 16:14:53,884 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 873
  total_bytes: 6338
}

2022-01-20 16:14:53,992 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 873
  total_bytes: 6338
}

2022-01-20 16:14:54,100 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 873
  total_bytes: 6338
}

2022-01-20 16:14:54,209 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6338
  total_bytes: 6338
}

2022-01-20 16:14:54,317 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6338
  total_bytes: 6338
}

2022-01-20 16:14:54,426 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6338
  total_bytes: 6338
}

2022-01-20 16:14:54,533 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6338
  total_bytes: 6338
}

2022-01-20 16:14:54,641 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6338
  total_bytes: 6338
}

2022-01-20 16:14:55,267 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6338
  total_bytes: 6338
}

2022-01-20 16:14:55,793 INFO    MainThread:25636 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6338
  total_bytes: 6338
}
local_info {
}

2022-01-20 16:14:57,094 INFO    MainThread:25636 [wandb_run.py:_append_files():2180] logging synced files
